uaclient/config.py

```diff
@@ -115,19 +115,21 @@ def entitlements(self):
 
         self._entitlements = {}
         contractInfo = machine_token["machineTokenInfo"]["contractInfo"]
-
+        tokens_by_name = dict(
+            # Handle Upper-case and lower-case keys until the following issue
+            # is fixed: ua-contracts #645
+            (e.get("Type", e.get("type")), e.get("Token", e.get("token")))
```

Comment:
I don't want to land this branch with this logic in place. We should work out what the contract server should be doing, get them to make the changes, and write against that.